src/test/java/org/apache/platypus/server/grpc/LuceneServerTest.java

```diff
@@ -466,6 +490,66 @@ public void testSearchDisjunctionMaxQuery() throws IOException, InterruptedExcep
         checkHits(hit);
     }
 
+    @Test
+    public void testSearchMatchQuery() throws IOException, InterruptedException {
+        GrpcServer.TestServer testAddDocs = new GrpcServer.TestServer(grpcServer, true, Mode.STANDALONE);
+        //2 docs addDocuments
+        testAddDocs.addDocuments();
+        //manual refresh
+        grpcServer.getBlockingStub().refresh(RefreshRequest.newBuilder().setIndexName(grpcServer.getTestIndex()).build());
+
+        SearchResponse searchResponse = grpcServer.getBlockingStub()
+                .search(SearchRequest.newBuilder()
+                        .setIndexName(grpcServer.getTestIndex())
+                        .setStartHit(0)
+                        .setTopHits(10)
+                        .addAllRetrieveFields(RETRIEVED_VALUES)
+                        .setQuery(Query.newBuilder()
```

Comment:
Maybe add tests for custom user input analyzer? Same for MatchPhraseQuery

Reply:
Added